README.md

```diff
@@ -32,6 +32,8 @@ a good network (which you can feed into this program, suddenly making it strong)
 
 # I want to help
 
+To contribute to leela-zero, you need to run the autogtp binary.
```

Comment:
This is explained below, I don't understand why this needs to be mentioned here if people don't even know what autogtp is yet.